test/test_browser.py

```diff
@@ -1881,6 +1881,22 @@ def setup():
     shutil.copy(Path('sub/test.data'), '.')
     self.btest_exit('test_emscripten_async_load_script.c', args=['-sFORCE_FILESYSTEM'])
 
+  @also_with_wasmfs
+  def test_emscripten_overlapped_package(self):
+    def setup():
+      create_file('script1.js', '''
+        Module._set(456);
```

Comment:
IIUC the test is testing for overlapping filenames. How is this `_set()` part helping?

Reply:
My bad, it’s a leftover from the async loading test I copied to get this test started.  I’ll reduce it.